core/src/test/scala/org/locationtech/rasterframes/functions/LocalFunctionsSpec.scala

```diff
@@ -38,10 +38,10 @@ class LocalFunctionsSpec extends TestEnvironment with RasterMatchers {
     it("should local_add") {
       val df = Seq((one, two)).toDF("one", "two")
 
-      val maybeThree = df.select(rf_local_add($"one", $"two")).as[ProjectedRasterTile]
-      assertEqual(maybeThree.first(), three)
+      val maybeThree = df.select(rf_local_add($"one", $"two")).as[Option[ProjectedRasterTile]]
+      assertEqual(maybeThree.first().get, three)
```

Comment:
I'm totally fine with this, but I'm afraid if @pomadchin ever sees this he'll call in the functional purity police 🚔 😆

Reply:
`.get` for the test only, no harm!

Reply:
@metasim @echeipesh for _tests_ only 👁️

Reply:
@pomadchin where's your work with using Frameless for encoding?

Reply:
@metasim diff is here https://github.com/echeipesh/rasterframes/pull/2/files, I think @echeipesh continues the work using my branch. But it's not pushed yet cc @echeipesh

Reply:
Awesome... Any chance the work could be coalesced into this PR? Nice to be able to see the progress and have a common view of where the current blockers are.

Reply:
Or is the issue here that that STAC DataSource is interleaved with this work?